engine/src/main/java/org/terasology/world/block/family/AbstractBlockFamily.java

```diff
@@ -29,13 +32,23 @@
     private BlockUri uri;
     private Set<String> categories = Sets.newHashSet();
 
-    public AbstractBlockFamily(BlockUri uri, Iterable<String> categories) {
-        this.uri = uri;
+    public AbstractBlockFamily(BlockFamilyDefinition definition, BlockShape shape, BlockBuilderHelper blockBuilder) {
```

Comment:
seems like these constructors should set `protected` properties on the base class. They're called via `super` below.